cattle/plugins/host-api/host_api.py

```diff
@@ -16,8 +18,12 @@ def on_startup(self):
                     '-cadvisor-url',  url,
                     '-logtostderr=true',
                     '-ip', Config.host_api_ip(),
-                    '-port', str(Config.host_api_port())],
+                    '-port', str(Config.host_api_port()),
+                    '-auth=true',
+                    '-host-uuid', DockerConfig.docker_uuid(),
+                    '-public-key', Config.jwt_public_key_file()],
                    env=env)
+        logging.info("Turning on JWT-Authentication")
```

Comment:
No need for this log statement, besides that fact the message is wrong.  What is being done here is that the Host API is being started.